subtests/docker_cli/run_attach/run_attach.py

```diff
@@ -53,7 +53,7 @@ def _init_container(self, tty, cmd, cmd_input='\n'):
         fin = DockerImage.full_name_from_defaults(self.config)
         subargs.append(fin)
         subargs.append(cmd)
-        cmd = DockerCmd(self, 'run', subargs, timeout=3, verbose=False)
+        cmd = DockerCmd(self, 'run', subargs, timeout=30, verbose=False)
```

Comment:
I agree, never liked those short timeouts.  They lead to quite a few "discussions" in the past.